globals-shelley-dev.nix

```diff
@@ -0,0 +1,49 @@
+self: super: {
+  globals = (import ./globals-defaults.nix self) // rec {
+
+    static = import ./static;
```

Comment:
can we rename this to `secrets`?

Reply:
since it is somewhat a separated issue and need a PR to ops-lib, I have open https://github.com/input-output-hk/ops-lib/issues/8 to track this.